common/scala/src/main/scala/whisk/common/ConsulClient.scala

```diff
@@ -55,18 +53,16 @@ object ConsulEntry extends DefaultJsonProtocol {
  * Client to access Consul's <a href="https://www.consul.io/docs/agent/http/kv.html">
  * key/value store</a>
  */
-class ConsulClient(host: String)(implicit val actorSystem: ActorSystem) {
+class ConsulClient(hostAndPort: String)(implicit val actorSystem: ActorSystem) {
```

Comment:
@ChrBi @markusthoemmes you may want to review this given your recent changes to `ConsulClient`